src/services/verifyToken.js

```diff
@@ -0,0 +1,24 @@
+/*
+Middleware to verify the JSON Web Token and validate the client
+*/
+import jsonwebtoken from 'jsonwebtoken';
+
+export default function verifyToken (req, res, next) {
+// Get the token from the HTTP headers
+  const SECRET_KEY = process.env.TOKEN_SIGNATURE || 'ssshhh';
```

Comment:
This is already defined here https://github.com/asylum-connect/catalog-api/blob/e2ef367e5b298ca1977075f8de146cd5e08447b4/src/utils.js#L4. Defining it in two places like this is dangerous as it could lead to the tokens falling out of sync in the future and adds more complexity to managing them. Can you either create a src/config.js file similar to what we use in the [catalog](https://github.com/asylum-connect/asylumconnect-catalog/blob/master/src/config.js) or [control panel](https://github.com/asylum-connect/control-panel/blob/master/src/utils/config.js) so that our config values are centralized and imported by the consuming files?

Reply:
I pushed some new changes to dev that now have a config file with this value. Please import it from there